packages/common-substrate/src/project/versioned/v0_3_0/model.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {ProjectManifestBaseImpl} from '@subql/common';
+import {
+  SubqlCustomDatasource,
+  SubqlCustomHandler,
+  SubqlMapping,
+  SubqlNetworkFilter,
+  SubqlRuntimeHandler,
+} from '@subql/types';
+import {plainToClass, Type} from 'class-transformer';
+import {Equals, IsArray, IsObject, IsOptional, IsString, ValidateNested, validateSync} from 'class-validator';
+import yaml from 'js-yaml';
+import {CustomDataSourceBase, Mapping, RuntimeDataSourceBase} from '../../models';
+import {
+  CustomDatasourceV0_3_0,
+  SubstrateProjectManifestV0_3_0,
+  RuntimeDataSourceV0_3_0,
+  SubqlMappingV0_3_0,
+} from './types';
+
+export class FileTypeV0_3_0 {
```

Comment:
There's no need to define a new class if the previous version is the same. We can reuse a lot of code. 

If you rebase you will see how v0.2.1 was done.